src/main/java/com/gd/clinic/entities/Patient.java

```diff
@@ -0,0 +1,53 @@
+package com.gd.clinic.entities;
+
+import lombok.*;
+import org.hibernate.Hibernate;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.util.Objects;
+import java.util.UUID;
+
+
+@RequiredArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "Patients")
```

Comment:
please call table from lowercase

Reply:
Corrected that this commit, will use lowercase from now on. This has been corrected for all entities.